docs/cloud/dbaas/backups.md

```diff
@@ -11,23 +11,23 @@ after the last backup. All backups are stored for 90 days after which they get a
 deleted (not yet implemented). In the future, it will only be possible to delete backups automatically,
 users will not be able to manually delete their own backups.
 
-The backups are stored encrypted in Allas. 
+The backups are stored encrypted in Allas.
 
 You can manage your backups from the CLI and the web-interface.
 
 ## Risks
 
 Be aware that the backup feature is still in development and should not be used for anything remotely sensitive or important.
 
-* Backups are stored encrypted in Allas. This is worth to keep in mind if you want to take your own backup (you should not store it in Allas).
+* Backups are stored encrypted in Allas. This is worth to keep in mind if you want to take your own backup (you should probably store them somewhere else than Allas nor cPouta).
 * When "restoring" a backup it will create a new instance with a new IP. So be aware that you might need to update your egress firewalls in your service.
 * If you have any concerns, do not hesitate to contact [CSC Service Desk](../../support/contact.md).
-* All your database data and backups are stored in the CSC Kajaani datacenter. If the datacenter burns down you will lose all your data.
+* All your database data and backups are stored in the CSC Kajaani datacenter. If the datacenter "implode" you will lose all your data.
```

Comment:
This should be "implodes", and probably be followed by a comma. And maybe a different expression:

```suggestion
* All your database data and backups are stored in the CSC Kajaani datacenter. If the datacenter has catastrophic data loss, you may lose all your data.
```

Reply:
Update as suggested.